test/spec/Feature/Query/PlanSpec.hs

```diff
@@ -254,7 +254,7 @@ spec actualPgVersion = do
           resStatus  = simpleStatus r
 
       liftIO $ do
-        resHeaders `shouldSatisfy` elem ("Content-Type", "application/vnd.pgrst.plan+text; charset=utf-8")
+        resHeaders `shouldSatisfy` elem ("Content-Type", "application/vnd.pgrst.plan; charset=utf-8")
```

Comment:
This one slipped me by. Will correct on https://github.com/PostgREST/postgrest/pull/2825